### PR TITLE
Devb

### PR DESCRIPTION
**Pull request by lovendatj**

### CODE REVIEW

File: src/components/lib/ProjectCard.tsx [hunk 1]
```diff
@@ -49,13 +57,13 @@ export const ProjectCard: FC<ProjectCardProps> = ({
 export const ProjectCards: FC = () => {
     const { execute, status, error, value: projects } = useGithubRepos({
         // eslint-disable-next-line @typescript-eslint/camelcase
-        per_page: 4,
+        per_page: num_cards,
```
Comment (DonIsaac) [inline src/components/lib/ProjectCard.tsx]: Make this configurable as a prop

File: src/components/Projects.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React, { FC } from "react"
+import { ProjectCards } from "./lib/ProjectCard"
+import "../assets/stylesheets/index.scss"
+
+export const Projects: FC = () => {
```
Comment (DonIsaac) [inline src/components/Projects.tsx:5]: Remove `{}` and `return`, they are not necessary

File: src/components/Main.tsx [hunk 3]
```diff
@@ -81,15 +81,6 @@ export const Main: FC = () => {
                     </div>
                 </div>
             </section>
-            <section id="projects" className="page-section">
```
Comment (DonIsaac) [inline src/components/Main.tsx:84]: The landing page should display the top 4 projects

**Review by DonIsaac — CHANGES_REQUESTED** [review]

File: src/components/Projects.tsx (hunk 2 above)
Comment (lovendatj) [reply src/components/Projects.tsx:5]: Remove what?

**Review by DonIsaac — APPROVED** [review]

**Pull request merged**